server/src/main/java/io/deephaven/server/console/ConsoleServiceGrpcImpl.java

```diff
@@ -189,7 +190,13 @@ public void executeCommand(
                             response))
                     .submit(() -> {
                         final ScriptSession scriptSession = exportedConsole.get();
-                        final ScriptSession.Changes changes = scriptSession.evaluateScript(request.getCode());
+
+                        final ScriptSession.Changes changes = request.getSystemic()
```

Comment:
If we use `optional` in the protobuf, I think the proper downstream usage should include a `request.hasSystemic` check, in which case this will looks something like `request.hasSystemic() && request.getSystemic()`.